### ISSUE TITLE
Voice messages cannot be played on mobile devices

### ISSUE THREAD

**Issue by jprusch** [streamer45/mattermost-plugin-voice#14]
Only the message "Voice message" is printed.

**Comment by robertoschiano68** [comment]
The same thing happens to me too. It is essential that it works in mobile apps ...

**Comment by overflood** [comment]
This is important. I even don't find the "record voice" on mobile.

**Comment by jprusch** [comment]
Right, with the newest mobile app on iOS I'm not able to record voice messages as on earlier releases. This used to be done by using the built in voice recording when choosing media.

**Comment by Rmanal** [comment]
Yes it is a major point if we can not record message or play them on mobile app.
Any workaround?

**Comment by streamer45** [comment]
As explained in #3 (comment) Mattermost native clients do not have support for client-side plugins. 
I'll update the README to include a note.

**Comment by jprusch** [comment]
Sehr geehrte Damen und Herren,
ich bin vom 8.6. bis zum 12.6. nicht im Hause und habe daher leider keinen Zugriff auf meine E-Mails.
Ihre Mail wird NICHT weitergeleitet und nach meiner Rückkehr bearbeitet.
In dringende Fällen wenden Sie sich bitte an Hr. Tarrach (v.tarrach@schaeferbarthold.de)
Mit freundlichen Grüßen

i.V. Jan-Peter Rusch

TEL +49 (0)571-502-145 | FAX +49 (0)571-502-66-145
j.rusch@schaeferbarthold.de
______________________________________________________________________
Schäferbarthold GmbH
Erbeweg 2-12 | 32457 Porta Westfalica | Deutschland
www.schaeferbarthold.de
Schäferbarthold GmbH: Amtsgericht Bad Oeynhausen HRB 3748
Geschäftsführer: Dr. Volker Schäferbarthold, Dr. Christian Schäferbarthold, Wolfgang Schäferbarthold
As explained in #3 (comment) Mattermost native clients do not have support for client-side plugins. 
I'll update the README to include a note.

**Comment by samirmhsnv** [comment]
after 2 years. any news ?)

**Comment by streamer45** [comment]
@samirmhsnv 

Yes, there are [concrete plans](https://mattermost.atlassian.net/browse/MM-38573) to integrate this functionality in Mattermost which also means mobile support. Can't speak of precise timelines yet but I am looking to work on this in the next couple of months.

**Comment by faris-mohamed10** [comment]
Any updates on this? Just enquiring so that if there is a scope for contribution we can look into it as well.
@streamer45

**Comment by streamer45** [comment]
@faris-mohamed10 No update unfortunately. The work I linked above was halted as we prioritized other functionalities. I am still planning to get back to it eventually but I am going to avoid raising hopes once more here because it's been years now. But I am very interested in contribution opportunities, please feel free to reach out on [our public server](https://community.mattermost.com/) and we can discuss further.

**Comment by hrkh1990** [comment]
@streamer45 please update this project ,  after 2 years of waiting .

**Comment by streamer45** [comment]
@hrkh1990 Nothing has changed really :p Again, it's an open-source repository (MIT licensed), so I encourage everyone to send pull requests if they have the time to improve things, and I'll be more than happy to review them.
